src/hooks/members/useMemberFilters.ts

```diff
@@ -34,14 +34,56 @@ const useMemberFilters = ({
 
   const { colonyAddress = '' } = colony ?? {};
 
+  const databaseDomainIds = useMemo(
+    () =>
+      new Set(
+        selectedTeam
+          ? [selectedTeam.id]
+          : nativeDomainIds.map((id) => `${colonyAddress}_${id}`),
+      ),
+    [nativeDomainIds, colonyAddress, selectedTeam],
+  );
+
+  // Always include the root domain, since if the user has a permission in root, they have it in all domains
+  const permissionsDomainIds = useMemo(
+    () => new Set([`${colonyAddress}_${Id.RootDomain}`, ...databaseDomainIds]),
+    [databaseDomainIds, colonyAddress],
+  );
```

Comment:
Could we use `getDomainDatabaseId` util here?

Reply:
And is the useMemo necessary?

Reply:
I think so, these const are being used by other parts of the code that are memorized so if I remove the `useMemo` here it tells me that it needs it to not re-render the other functions.

Reply:
Fair enough if that's the case, although I bet wherever the "useMemo chain" starts it's not needed there either